### PR TITLE
Meetup models

### PR DESCRIPTION
**Pull request by exploreshaifali**
Meetup model and its test.

### CODE REVIEW

File: systers_portal/meetup/models.py [hunk 1]
```diff
@@ -21,3 +23,24 @@ class MeetupLocation(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Meetup(models.Model):
+    """Manage details of Meetups of MeetupLocations"""
+    title = models.CharField(max_length=50, verbose_name="Title",)
+    slug = models.SlugField(max_length=150, unique=True, verbose_name="Slug")
```
Comment (ana-balica) [inline systers_portal/meetup/models.py]: Why is slug max_length much bigger than the title length?

File: systers_portal/meetup/models.py [hunk 2]
```diff
@@ -21,3 +23,24 @@ class MeetupLocation(models.Model):
...
+    date = models.DateField(verbose_name="Date")
+    time = models.TimeField(verbose_name="Time", blank=True)
+    venu = models.TextField(verbose_name="Venu", blank=True)
```
Comment (ana-balica) [inline systers_portal/meetup/models.py]: venue*

File: systers_portal/meetup/models.py [hunk 3]
```diff
@@ -21,3 +23,24 @@ class MeetupLocation(models.Model):
...
+    description = RichTextField(verbose_name="Description")
+    meetup_location = models.ForeignKey(MeetupLocation, verbose_name="Meetup Location")
+    creator = models.ForeignKey(SystersUser, verbose_name="Creator")
```
Comment (ana-balica) [inline systers_portal/meetup/models.py]: `creator` sounds a bit unusual. Maybe better `created_by`?

File: systers_portal/meetup/tests/test_models.py [hunk 4]
```diff
@@ -1,10 +1,15 @@
+from django.contrib.auth.models import User
 from django.test import TestCase
+from django.utils import timezone
+from django.core.exceptions import ValidationError
```
Comment (ana-balica) [inline systers_portal/meetup/tests/test_models.py]: Alphabetical order.

File: systers_portal/meetup/tests/test_models.py [hunk 5]
```diff
@@ -12,8 +17,36 @@ def setUp(self):
...
+                                            venu="FooBar colony",
+                                            description="This is a testing meetup.",
+                                            meetup_location=self.meetup_location,
+                                            creator=self.systers_user)
+
+    def test_str(self):
+        """Test Meetup object str/unicode representation"""
+        self.assertEqual(str(self.meetup), "Test Meetup")
+
+    def test_meetup_date(self):
```
Comment (ana-balica) [inline systers_portal/meetup/tests/test_models.py]: Looks good :)

File: systers_portal/meetup/migrations/0002_meetup.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        ('meetup', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Meetup',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('title', models.CharField(max_length=50, verbose_name='Title')),
+                ('slug', models.SlugField(verbose_name='Slug', unique=True)),
```
Comment (ana-balica) [inline systers_portal/meetup/migrations/0002_meetup.py]: I see you updated the migrations. Just curious why the slug doesn't have the `max_length` restriction...

Comment (exploreshaifali) [inline systers_portal/meetup/migrations/0002_meetup.py]: I think it was by mistake, it should be 50, I checked in model also slug does not have any `max_length` restriction that is why migrations also not showing it. I will update model as well as migration.

**Pull request merged**